app/models/driver.rb

```diff
@@ -3,6 +3,6 @@ class Driver < ApplicationRecord
   validates :license_number, presence: true
   validates :mobile_number, presence: true, phone: true
   validates :email_address, presence: true, email: true
-  validates :age, presence: true
-  validates :status, presence: true
+  validates :age, presence: true, numericality: { greater_than: 0}
```

Comment:
```suggestion
  validates :age, presence: true, numericality: { greater_than_equal_to: 18, less_than_or_equal_to: 74 }
```
This would then match singapore driver laws.
then we can update the tests to ensure this validation means we cant make 1 year old drivers

Reply:
was this change made?
please resolve *after* changes are made